recipes/webdavclient2/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "webdavclient2" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "797f121c4e381cb57d7f46abef513191da74959d70096e435269d9479a71ee7f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - requests
+    - lxml
+    - argcomplete
+
+test:
+  commands:
+    # script currently broken, issue #4
```

Comment:
Should we comment `commands` then also as it is empty? Also do you mind adding the full link to the issue for clarity?

Reply:
have not thought on just replying here. @dougalsutherland mentioned the command and by that I found the typo.